src/backend/api/mirrors_update.py

```diff
@@ -269,18 +271,19 @@ async def mirror_available(
                     err,
                 )
                 return mirror_name, False
-    # if mirror has no valid version/arch combos it is dead
-    logger.error(
-        'Mirror "%s" has no valid repositories',
-        mirror_name
+    # if mirror has at least one valid version/arch combo
```

Comment:
We loop through all versions and arches and we think a mirror is valid when it has all version/arches.